elkm1_lib/message.py

```diff
@@ -484,6 +492,11 @@ def ka_encode() -> MessageEncode:
     return MessageEncode("06ka00", "KA")
 
 
+def kf_encode(keypad: int, key: str=FunctionKeys.NULL.value) -> MessageEncode:
```

Comment:
This is the only encode function that has a default value. What's the thinking here? 

Change type to `FunctionKeys` and then people are forced to pass in a valid function key.

Reply:
I added the default so keypads->sync could just call with no arguments and not need to know that null should be sent:
https://github.com/jimboca/elkm1/blob/master/elkm1_lib/keypads.py#L51
Will change the type.